syncd/syncd.cpp

```diff
@@ -2356,13 +2357,25 @@ void processPfcWdEvent(
 
         if (op == DEL_COMMAND)
         {
-            PfcWatchdog::removeQueue(queueVid);
-            continue;
+            if (objectType == SAI_OBJECT_TYPE_PORT)
+            {
+                PfcWatchdog::removePort(vid);
+                continue;
+            }
+            else if (objectType == SAI_OBJECT_TYPE_QUEUE)
+            {
+                PfcWatchdog::removeQueue(vid);
+                continue;
+            }
+            else
+            {
+                SWSS_LOG_ERROR("Object tyoe for removal not supported");
```

Comment:
tyoe -> type

Reply:
here you also need continue.

I think it is better to do as follows, and it will be more clear.

if (op == DEL_COMMAND)
{
}
else if (op == SET_COMMAND)
{
}
else
{
}

Reply:
Done